runtime/interpreter/encode.go

```diff
@@ -1052,22 +1267,36 @@ const (
 	//
 	// encodedCompositeStaticTypeLength MUST be updated when new element is added.
 	// It is used to verify encoded composite static type length during decoding.
-	encodedCompositeStaticTypeLength int = 3
+	encodedCompositeStaticTypeLength = 3
 )
 
-func (e *Encoder) prepareCompositeStaticType(v CompositeStaticType) (interface{}, error) {
-	location, err := e.prepareLocation(v.Location)
+// encodeCompositeStaticType encodes CompositeStaticType as
```

Comment:
Thanks for adding all this documentations. Makes it really easy to understand what the code does / encoding schema.

Reply:
💯